quidel_covidtest/delphi_quidel_covidtest/run.py

```diff
@@ -55,7 +56,7 @@ def get_smooth_info(sensors, _SMOOTHERS):
             smoothers[sensor] = smoothers.pop(RAW_TEST_PER_DEVICE)
     return smoothers
 
-def run_module(params: Dict[str, Any]):
+def run_module(params: Dict[str, Any]): # pylint: disable=too-many-statements
```

Comment:
(54 statements rather than the maximum 50)

I'm open to splitting this function into multiple helpers if you think that's a better option, though :)